tensorflow_encrypted/tensor/native.py

```diff
@@ -7,6 +7,8 @@
 from ..config import run
 
 INT_TYPE = tf.int32
+bits = 31
+p = 67
 
 
 class NativeTensor(object):
```

Comment:
Should NativeTensor be an abstract base class here? Unless there's a good reason for the opposite, it's usually good practice to do so.

Reply:
Do you have a good link explaining the benefits of ABC? I can't find anything concrete. I'm sure this can just be one though.

Reply:
As far as I can ABCs shouldn't have implementation.

Reply:
I'm going to merge this for now can figure it out later. I think this might come into play in the PR i'm working on right now.